test/instrumentation/decode_metric.go

```diff
@@ -182,7 +183,24 @@ func (c *metricDecoder) decodeOpts(expr ast.Expr) (metric, error) {
 				if err != nil {
 					return m, err
 				}
+			case *ast.SelectorExpr:
+				packageName := fmt.Sprintf("%v", v.X)
+
+				variableExpr, found := c.variables[packageName + "." + v.Sel.Name]
+				if !found {
+					return m, newDecodeErrorf(expr, errBadVariableAttribute)
+				}
+				bl, ok := variableExpr.(*ast.BasicLit)
+				if !ok {
+					return m, newDecodeErrorf(expr, errNonStringAttribute)
+				}
+				value, err = stringValue(bl)
+				if err != nil {
+					return m, err
+				}
+
 			default:
+				fmt.Fprintf(os.Stdout, "key %s is type %T from Default\n", key, v)
```

Comment:
i think we have to log this directly into a file, i suspect we rely on stdout to output the stuff into yaml

Reply:
removed debug line